src/shared/common/BottomProgress.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from 'react';
-import * as Progress from 'react-native-progress';
+import ProgressBar from 'react-native-progress/Bar';
```

Comment:
Is there a good way to check for things like this? Or just trying the app and noticing that some errors will occur?

Reply:
check the release notes, that's the ideal way :)

Reply:
okay good.